crates/vizia_core/src/view.rs

```diff
@@ -28,6 +32,21 @@ pub trait View: 'static + Sized {
         cx.cache.add(id).expect("Failed to add to cache");
         cx.style.add(id);
         cx.views.insert(id, Box::new(self));
+        let parent_id = cx.tree.get_layout_parent(id).unwrap();
+        let parent_node_id = parent_id.accesskit_id();
+        let parent_node = cx.get_node(parent_id);
+        let node_id = id.accesskit_id();
+        let children =
+            parent_id.child_iter(&cx.tree).map(|entity| entity.accesskit_id()).collect::<Vec<_>>();
+
+        cx.tree_updates.push(TreeUpdate {
```

Comment:
Potentially... this is updating the accessibility tree when a view is created, which occurs when `build` is called. More specifically this is updating the structure of the tree rather than the values on the nodes.